tests/test_bundletester_parallel.groovy

```diff
@@ -0,0 +1,51 @@
+def tests = ['test_bundletester']
```

Comment:
Where are the test_X_parallel.groovy used? I though we agreed we will be calling test_cdk_parallel.